opennopd/subsystems/climanager.c

```diff
@@ -65,6 +65,8 @@ void start_cli_server() {
         if (pthread_create(&sniffer_thread, NULL, client_handler,
                            (void*) new_sock) < 0) {
             perror("could not create thread");
+	    free(new_sock);
+	    new_sock = NULL;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "new_sock". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=c%3AS1854)